kythe/go/serving/tools/write_tables/write_tables.go

```diff
@@ -139,12 +140,23 @@ func main() {
 	}
 
 	if *compactTable {
```

Comment:
Theres a call to compactleveldb at line 94, too. does that need to be updated?

Reply:
This write_tables script really has two codepaths: 1 that uses apache beam (flume basically) and one that does not. I've modifed the non-beam codepath so it supports pebble, but the beam one still requires more work due to how it writes the output files.